qa/crontab/teuthology-cronjobs

```diff
@@ -111,17 +110,15 @@ TEUTHOLOGY_SUITE_ARGS="--non-interactive --newest=100 --ceph-repo=https://git.ce
 24 00 * * 1        $CW $SS 120000 --ceph quincy --suite upgrade:octopus-x -p 820
 32 00 * * 1        $CW $SS 120000 --ceph quincy --suite upgrade:pacific-x -p 820
 
-### upgrade runs for reef release
+### upgrade runs for tentacle release
 ###### on smithi
 
-08 01 * * 3        $CW $SS      1 --ceph reef --suite upgrade:pacific-x -p 830
-16 01 * * 3        $CW $SS      1 --ceph reef --suite upgrade:quincy-x  -p 830
+08 01 * * 3        $CW $SS      1 --ceph tentacle --suite upgrade -p 100
 
 ### upgrade runs for squid release
 ###### on smithi
 
-#                                                              -p 840
-08 02 * * 5        $CW $SS     32 --ceph squid --suite upgrade -p 100 --force-priority
+08 02 * * 5        $CW $SS     32 --ceph squid --suite upgrade -p 840 --force-priority
```

Comment:
Is the goal here just to change priority?

Reply:
Yes